### PR TITLE
Fix yaml-lint everywhere

### PR DESCRIPTION
**Pull request by Tompage1994**
## What this fixes
The github action is failing because of failing `yaml-lint` tests. This should fix all of them.

This also renames a load of files `*.yaml` > `*.yml` as per the repo standards.

Sorry for the **_HUGE_** diff

### NOTE
The `ansible-lint`ing will be much more involved so I have not made a start on that.

### CODE REVIEW

File: .yamllint.yml [hunk 1]
```diff
@@ -2,6 +2,5 @@ extends: default
 
 rules:
   # 80 chars should be enough, but don't fail if a line is longer
-  line-length:
-    max: 80
-    level: warning
+  line-length: disable
+  colons: disable
```
Comment (kedark3) [inline .yamllint.yml:6]: Kindly add the new line at the end of this file so it doesn't show the warning like this: 
image

Comment (Tompage1994) [reply .yamllint.yml]: Have updated the yaml-lint to check this file too

**Comment by Tompage1994** [comment]
> @Tompage1994 thanks for the PR. I will review it. I was going to work on it, but if you'd like to continue with ansible-lint please let me know, I will work on other stuff I have to get in :)

I have been working on this this evening. I'm almost done. Do you reckon I should add the changes into this PR or raise a separate one to keep reviews easier?

**Comment by kedark3** [comment]
@Tompage1994 please add ansible-lint changes to different PR :) Thank you.

**Comment by kedark3** [comment]
@Wilk42 I approved this PR, would you like to review it before merge? It is straightforward :100:

**Pull request merged**